Yank/pipeline.py

```diff
@@ -281,7 +281,7 @@ def prepare_phase(positions_file_path, topology_file_path, ligand_dsl, system_op
             err_msg = 'Found periodic box in inpcrd file but nonbondedMethod is NoCutoff'
     else:
         if system_options['nonbondedMethod'] != openmm.app.NoCutoff:
```

Comment:
This should check for all of the periodic nonbonded methods, right?
```python
periodic_nonbonded_methods = [openmm.app.CutoffPeriodic, openmm.app.PME, openmm.app.Ewald]
if system_options['nonbondedMethod'] in periodic_nonbonded_methods:
```
Otherwise, we may miss `CutoffNonPeriodic`, which we don't want to recommend but should still be allowed for benchmarking purposes.